internal/controller/api/job_receiver.go

```diff
@@ -76,36 +78,42 @@ func (jr *JobReceiver) handleJob() http.HandlerFunc {
 
 	return func(w http.ResponseWriter, req *http.Request) {
 
-		log.Println("Simulating JobReceiver producing a message")
+		principal, _ := middlewares.GetPrincipal(req.Context())
```

Comment:
Just a note, but I probably should have put the auth middleware into its own package.

Reply:
We can always move code around later.